raven/raven.go

```diff
@@ -80,6 +80,9 @@ type sentryResponse struct {
 	ResultId string `json:"result_id"`
 }
 
+// Default sentry DSN from https://github.com/getsentry/sentry-java/blob/4b810150969108429bbb36d28227cb128b986aa0/sentry/src/main/java/io/sentry/dsn/Dsn.java#L21
+const DefaultSentryDSN = "noop://user:password@localhost:0/0"
```

Comment:
This looks different from the value in the link:

```
public static final String DEFAULT_DSN = "noop://localhost?async=false";
```

Is there a reason they can't be identical?

Reply:
That's interesting! I'm not sure why, but I saw the value as the older one, and now I'm scratching my head figuring out why that was.

In any regard, thanks for clicking on the link and I'll update the value in the next commit.